modules/nf-core/ampcombi/main.nf

```diff
@@ -0,0 +1,48 @@
+process AMPCOMBI {
+    tag "$meta.id"
+    label 'process_medium'
```

Comment:
```suggestion
    label 'process_single'
```

No threading (although that would probably be cool for the parsing step, if the tool can be slow)

Reply:
We put medium because of the alignment step with Diamond, which requires threading. (There is a diamond module as well and it has 'process_medium' as well

Reply:
Ok but where do you supply the threads to ampCOMBI to give to DIAMOND? I don't see any flag for this in the github repo/help? https://github.com/Darcy220606/AMPcombi

Reply:
Alright we added the threads fro diamond in the new ampcombi release